### PR TITLE
Increase travis timeout to 20 minutes

### PR DESCRIPTION
**Pull request by nikitin-da**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jdk: oraclejdk8
 before_install:
   - sudo apt-get update -q
   - sudo apt-get install -y curl unzip
-  - pip install --user codecov
+  - travis_wait 20 pip install --user codecov
```
Comment (artem-zinnatullin) [inline .travis.yml]: wut, do we actually spend time on this?

Comment (nikitin-da) [reply .travis.yml]: rtfm

**Comment by artem-zinnatullin** [comment]
```console
com.pushtorefresh.storio2.sqlite.integration.RxQueryTest > queryOneExistedObjectTableUpdate PASSED

travis_time:end:0989402b:start=1504075636448415121,finish=1504076838809435674,duration=1202361020553
[0K
[31;1mThe command "travis_wait 20 sh ci.sh" exited with 137.[0m
```

Code 137, looks like we are hitting memory limit :(

**Comment by geralt-encore** [comment]
and this one can be closed as well?

**Comment by nikitin-da** [comment]
Closed via #829

**Pull request closed without merging**